packages/ts-config/src/cjs/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+	"compilerOptions": {
+		"module": "CommonJS",
```

Comment:
nononononononononono, we should NOT be doing this. It should be the same as ESM, otherwise TS will not know about exports stuff.